src/Utils/queryUtils.js

```diff
@@ -0,0 +1,22 @@
+const applyFilters = (query, filters) =>
```

Comment:
I think this function should be renamed something like `pipeFilters` to make it more consistent with funcional programming naming conventions

Reply:
I think you are right, but i also think that `applyFilters` gives a better description of what this function does. And it makes it easier for a dev to understand the code.